ffs/scheduler/scheduler.go

```diff
@@ -422,6 +408,17 @@ func (s *Scheduler) executeQueuedJob(j ffs.Job) {
 		cancel()
 	}()
 
+	// Get
+	a, err := s.as.Get(j.ID)
+	if err != nil {
+		log.Errorf("getting push config action data from store: %s", err)
+		if err := s.js.Finalize(j.ID, ffs.Failed, err, nil); err != nil {
+			log.Errorf("changing job to failed: %s", err)
+		}
+		s.l.Log(ctx, a.Cfg.Cid, "Job %s couldn't start: %s.", j.ID, err)
+		return
+	}
```

Comment:
Moved from above, plus finalizing the job as a failure, since we can't do anything (isn't a retry issue).